main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -51,6 +51,9 @@ class Flix {
     */
   private val internals = List(
     "Prelude.flix" -> StreamOps.readAll(LocalResource.Library.Prelude),
+    "Option.flix" -> StreamOps.readAll(LocalResource.Library.Option),
```

Comment:
Is it preferred to include Libraries here, or in FlixTest.scala and other testing suites when needed?